### ISSUE TITLE
Syntax Error on If .... Then

### ISSUE THREAD

**Issue by vbasky** [TwitchBronBron/vscode-brightscript-language#101]
Valid syntax now shows expected a statement to follow if ... condition

<img width="552" alt="screen shot 2019-03-05 at 10 19 21 am" src="https://user-images.githubusercontent.com/287302/53769706-d79ecf00-3f30-11e9-9f5a-4ba440c817b9.png">

Version: 1.9.0-beta5

**Comment by TwitchBronBron** [comment]
I'm trying to reproduce this, but it seems to be working for me. Can you try to narrow down the actual syntax causing the error? I'm thinking there might be a syntax bug somewhere else, causing a chain of additional syntax errors shown in your code. 

```brightscript
'This shows no syntax errors at all
sub Main()
    k = true
    if k then
        If true then
            m.play_button.setFocus(true)
        End If
    end if
end sub
```

**Comment by TwitchBronBron** [comment]
Nevermind! I just realized the issue is the capital `"T"` in `"Then"`. Thanks.

**Comment by TwitchBronBron** [comment]
I just opened [this issue](https://github.com/sjbarag/brs/issues/187) in the `brs` project. When it gets fixed there, I will update this project.

**Comment by vbasky** [comment]
Oh yeah the setting on vscode is 
```"brightscript.format.keywordCase": "title",````

**Comment by TwitchBronBron** [comment]
That's only used when you use the format document command in vscode. Shouldn't have any affect on the code itself (except for this bug lol)

**Comment by TwitchBronBron** [comment]
This was fixed in 1.9.0-beta7.
